### PR TITLE
Conda setup for CI

### PR DESCRIPTION
**Pull request by rth**
Following discussion in https://github.com/pytest-dev/pytest-qt/issues/180  this PR proposes a Travis CI setup using conda. Please let me know if I missed anything.

**Edit:** with the appveyor.yml addition this fixes https://github.com/pytest-dev/pytest-qt/issues/180

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,45 +3,48 @@ sudo: required
...
- - sudo $PYTHON get-pip.py
+ # Install 
+ - git clone --depth 1 git://github.com/astropy/ci-helpers.git
+ - export SETUP_XVFB=true && source ci-helpers/travis/setup_conda.sh
 
  # Dependencies
- - sudo $PYTHON -m pip install -U pytest tox coveralls
+ - |
+     if [[ ${PYTEST_QT_API} == 'pyside' ]]; then
+         pyqt_package="pyside==1.2.4"
```
Comment (nicoddemus) [inline .travis.yml]: What do you think about creating a `PYQT_PACKAGE` environment variable in the `env` section along with the others? There will be a small repetition, but I think that's OK.

Comment (nicoddemus) [inline .travis.yml]: I think we can use fuzzy versions like `pyside=1.*`, `pyqt=4.*` and `pyqt=5.*`.

**Comment by nicoddemus** [comment]
Also please make sure to add yourself as a contributor in the `README`.

**Comment by rth** [comment]
@nicoddemus Thanks a lot for the review! I addressed your comments and added the Appveyor setup (largely inspired by the one in qtpy). Also refactored a bit travis setup to be more consistent. Tests now pass. The only point is that in Travis there is a warning/error at the very end,
```
Done. Your build exited with 0.
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":99.0"
      after 2266 requests (2266 known processed) with 0 events remaining.
```
but I guess since tests pass that's not relevant...

**Comment by rth** [comment]
Also there is no linting in Appveyor, but since it's alredy done in Travis that should, hopefully be sufficient..

File: appveyor.yml [hunk 2]
```diff
@@ -1,27 +1,60 @@
...
+      CONDA_DEPENDENCIES: "pytest tox coveralls pyqt=4.*"
+    - PYTHON_VERSION: "3.4"
+      PYTEST_QT_API: "pyqt4v2"
+      CONDA_DEPENDENCIES: "pytest tox coveralls pyqt=4.*"
+    - PYTHON_VERSION: "3.4"
+      PYTEST_QT_API: "pyside"
+      CONDA_DEPENDENCIES: "pytest tox coveralls pyside=1.*"
+    - PYTHON_VERSION: "3.4"
+      PYTEST_QT_API: "pyqt5"
+      CONDA_DEPENDENCIES: "pytest tox coveralls pyqt=5.*"
```
Comment (nicoddemus) [inline appveyor.yml]: We are not using `tox` and `coveralls` here so I guess we can remove them.

**Comment by rth** [comment]
Thanks, you are right - I removed tox and coveralls from the Appveyor setup..

**Comment by nicoddemus** [comment]
Awesome, thank you, really appreciate it! 👍

**Pull request merged**